### PR TITLE
Test the CORS feature

### PR DESCRIPTION
**Pull request by TejasC88**
Polarion ID : CEPH-10355

Test CORS using cURL GET,PUT and DELETE api's
Pass log : http://magna002.ceph.redhat.com/cephci-jenkins/tchandra/CORS.txt

### CODE REVIEW

File: rgw/v2/tests/curl/configs/test_cors_using_curl.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# script: test_cors_using_curl.py
+# polarion: CEPH-10355
+config:
```
Comment (anrao19) [inline rgw/v2/tests/curl/configs/test_cors_using_curl.yaml:3]: description has polarion id : CEPH-8357557

File: rgw/v2/tests/curl/test_cors_using_curl.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
+"""
+Usage: test_cors_using_curl.py -c <input_yaml>
+polarion: CEPH-10355
```
Comment (anrao19) [inline rgw/v2/tests/curl/test_cors_using_curl.py:3]: Missmatch in polarion ID

File: rgw/v2/tests/curl/test_cors_using_curl.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+            with open(file_name, "a") as f:
+                f.write(cors_policy)
+            aws_reusable.put_bucket_cors(cli_aws, bucket_name, file_name, endpoint)
+            cors_origin = config.test_ops["cors_origin"]
+            log.info(f"Origin : {cors_origin}")
+            log.info("Test cURL PUT,GET and DELETE with mentioned origin")
+            # create objects
+            objects_created_list = []
+            # uploading data
+            log.info("s3 objects to create: %s" % config.objects_count)
```
Comment (anrao19) [inline rgw/v2/tests/curl/test_cors_using_curl.py:78]: Can we use f-string for formatting

File: rgw/v2/tests/curl/configs/test_cors_using_curl.yaml (hunk 1 above)
Comment (TejasC88) [reply rgw/v2/tests/curl/configs/test_cors_using_curl.yaml:3]: @anrao19 the description was wrong , updated it, thanks

File: rgw/v2/tests/curl/test_cors_using_curl.py (hunk 2 above)
Comment (TejasC88) [reply rgw/v2/tests/curl/test_cors_using_curl.py:3]: Description was wrong

File: rgw/v2/tests/curl/test_cors_using_curl.py [hunk 4]
```diff
@@ -0,0 +1,166 @@
+"""
+Usage: test_cors_using_curl.py -c <input_yaml>
+polarion: CEPH-10355
+<input_yaml>
+    Note: Following yaml can be used
+    test_cors_using_curl.yaml
+    
+Operation:
+    Test CORS on a bucket using CURL API calls.
```
Comment (viduship) [inline rgw/v2/tests/curl/test_cors_using_curl.py:9]: @TejasC88 Request to make the operation more descriptive step by step would be better

**Review by anrao19 — APPROVED** [review]
LGTM

File: rgw/v2/tests/curl/configs/test_cors_using_curl.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  user_count: 1
+  bucket_count: 1
+  objects_count: 1
```
Comment (hmaheswa) [inline rgw/v2/tests/curl/configs/test_cors_using_curl.yaml:6]: @TejasC88 
could you please increase objects count to something like 25

**Comment by ckulal** [comment]
please resolve merge conflict.

**Pull request closed without merging**